### ISSUE TITLE
Low prio: Pairing between two usb adpaters on the same system fails

### ISSUE THREAD

**Issue by infirit** [blueman-project/blueman#476]
I doubt many people will even attempt this so this is a low priority thing to fix.

I do get __one__ (no more) notification to confirm the pair, when I confirm it fails.

**Comment by cschramm** [comment]
Works for me in 2.0.3 ... at least once. :laughing:

**Comment by infirit** [comment]
Same behaviour for me under 2.0 branch, [full log](https://gist.github.com/infirit/714c90caa51669a0206a).

**Comment by cschramm** [comment]
Still relevant?

**Comment by infirit** [comment]
Yes, but **very** low priority as it does not affect 99.9% of users. (just silly contributors doing silly things like me). I'll assign it to myself but its not on my poke list for the weekend :wink:
